### PR TITLE
Openvino tensor representation documentation

### PR DESCRIPTION
**Pull request by szabi-luxonis**
Added support for Ubuntu 20.04 (ruby 2.7.0).
Added documentation for new API.

https://deploy-preview-93--nifty-bartik-6a9295.netlify.app/api/

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -409,9 +404,207 @@ Neural network results packet. It's not a single result, but a batch of results
...
+    Can be used if in [blob config file](#creating-blob-configuration-file) `output_format` is set to `raw`.
+    It returns a shaped numpy array for the specific network output tensor, based on the neural network's output layer information. 
+    
+    For example: in case of `Mobilenet-SSD` it returns a `[1, 1, 100, 7]` shaped array, where `numpy.dtype` is `float16`.
+    
+    Example of usage:
+    `nnetpacket.get_tensor(0)` or `nnetpacket.get_tensor('detection_out')`
+
+{: #nnetpacket_getitem}
+* [__[ Union[int, str] ]__](#nnetpacket_getitem) -> numpy.ndarray
```
Comment (VanDavv) [inline api.md:416]: Method name is missing
image

File: api.md [hunk 2]
```diff
@@ -409,9 +404,207 @@ Neural network results packet. It's not a single result, but a batch of results
...
+    
+    Returns TensorInfo in a dictionary where the `key` is the name of attribute.
+
+{: #get_dimension}
+* [__get_dimension__](#get_dimension)() -> [depthai.TensorInfo.Dimension](#Dimension)
     
+    Returns the specific dimension of the tensor, for example: `tensor_info.get_dimension(depthai.TensorInfo.Dimension.WIDTH)` returns the `WIDTH` of tensor.
+
+
+{: #Detections}
```
Comment (VanDavv) [inline api.md:511]: I would skip the Containers classes. They behave like lists and serve a similar purpose, so I think we're fine just skipping them as there's nothing special about them

File: api.md [hunk 3]
```diff
@@ -613,128 +806,91 @@ We've documented example usage of these compilers [here](https://github.com/luxo
     
 ### Creating Blob configuration file
     
-Config file is required to create a mapping between CNN output and Python API on host side. 
-Basically, whole configuration resolves around `tensors` array. One tensor represents one CNN output, so usually
-you'll have just one object in there, but if you'd like to use e.x. [age gender recognition](https://docs.openvinotoolkit.org/latest/omz_models_intel_age_gender_recognition_retail_0013_description_age_gender_recognition_retail_0013.html)
-you'll need to define two tenors
+If config file is emitted there is no decoding done on device <==> `output_format` is set to `raw`. The decoding must be done on host side, by the user.
```
Comment (VanDavv) [inline api.md:809]: typo? Should it be `omitted`?

**Review by VanDavv — CHANGES_REQUESTED** [review]

File: api.md (hunk 1 above)
Comment (szabi-luxonis) [reply api.md:416]: It's the operator [], so nnet_packet[0] is equivalent to nnet_packet.get_tensor(0)

Comment (VanDavv) [reply api.md:416]: Oh, ok. So maybe just name it ]`__getitem__`](https://docs.python.org/3/reference/datamodel.html#object.__getitem__)?
I was a bit confused seeing this and was not aware that it resolves to `[]`

File: api.md (hunk 2 above)
Comment (szabi-luxonis) [reply api.md:418]: We still need to document it, otherwise people won't know that it behaves like a container class.

**Review by VanDavv — APPROVED** [review]

**Pull request merged**